daemon/api_test.go

```diff
@@ -202,6 +202,7 @@ func (s *apiSuite) TestSnapInfoOneIntegration(c *check.C) {
 			"update_available":   "v2",
 			"rollback_available": "v0",
 			"channel":            "stable",
+			"price":              float64(0),
```

Comment:
I guess I could use something like: https://github.com/shopspring/decimal

Fortunately this number is only used for display purposes (comes through as a Javascript numeric type)